internal/pkg/plugininstaller/ci/installer.go

```diff
@@ -0,0 +1,62 @@
+package ci
+
+import (
+	"errors"
+
+	"github.com/devstream-io/devstream/internal/pkg/plugininstaller"
+	"github.com/devstream-io/devstream/pkg/util/git"
+)
+
+func PushCIFiles(options plugininstaller.RawOptions) error {
+	opts, err := NewOptions(options)
+	if err != nil {
+		return err
+	}
+	// 1. get git content by config
+	gitMap, err := opts.buildGitMap()
+	if err != nil {
+		return err
+	}
+	if len(gitMap) == 0 {
```

Comment:
duplicate judgment, only need to judge `err`. In the function of `getFromXXX`, once the error should `return nil,err`. Otherwise, when the upstream does not judge `err` directly when using map will not appear panic, resulting in very difficult to debug later.